ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-stable-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -46,6 +46,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: "4.17"
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks